lib/validate.js

```diff
@@ -89,7 +90,7 @@ module.exports = function makeValidatorMiddleware (middleware, schema, opts) {
       ajv = new Ajv({
         coerceTypes: opts.coerce === 'false' ? opts.coerce : true,
         removeAdditional: typeof opts.removeAdditional !== 'undefined' ? opts.removeAdditional : false,
-        extendRefs: typeof opts.extendRefs !== 'undefined' ? opts.extendRefs : undefined
+        strict: opts.strict === true ? opts.strict : false
```

Comment:
Minor comment, but is this just converting a boolean or undefined/null into a boolean or are there other edge cases? (I guess if this is JS, then you can realistically pass literally anything)

I was going to suggest using the boolean constructor, but that might have unintended consequences if you pass, say, the number 5.

Reply:
Oh, and also: what is this option used for? I don't see it referenced anywhere in the PR. What does it do?